providers/aws/services/accessanalyzer/accessanalyzer_service.py

```diff
@@ -0,0 +1,117 @@
+import threading
+from dataclasses import dataclass
+
+from lib.logger import logger
+from providers.aws.aws_provider import generate_regional_clients
+
+
+################## AccessAnalyzer
+class AccessAnalyzer:
+    def __init__(self, audit_info):
+        self.service = "accessanalyzer"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.regional_clients = generate_regional_clients(self.service, audit_info)
+        self.analyzers = []
+        self.__threading_call__(self.__list_analyzers__)
+        self.__list_findings__()
+
+    def __get_session__(self):
+        return self.session
+
+    def __threading_call__(self, call):
```

Comment:
We need to check if we can work with `ThreadPoolExecutor()` which seems to be the recommended way to handle pools of threads in Python